standalone-metastore/metastore-server/pom.xml

```diff
@@ -44,6 +44,16 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.apache.datasketches</groupId>
+      <artifactId>datasketches-hive</artifactId>
+      <exclusions>
```

Comment:
could we declare the exclusions in mvn dependency management so we won't have to define them every time we use the dependency?